static/app/views/issueDetails/quickTrace/issueQuickTrace.tsx

```diff
@@ -1,89 +1,110 @@
+import {useEffect} from 'react';
 import styled from '@emotion/styled';
 import {Location} from 'history';
 
-import {Alert} from 'sentry/components/alert';
-import {Button} from 'sentry/components/button';
 import ErrorBoundary from 'sentry/components/errorBoundary';
 import ExternalLink from 'sentry/components/links/externalLink';
 import QuickTrace from 'sentry/components/quickTrace';
-import {IconClose} from 'sentry/icons';
+import {
+  ErrorNodeContent,
+  EventNode,
+  QuickTraceContainer,
+  TraceConnector,
+} from 'sentry/components/quickTrace/styles';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconFire} from 'sentry/icons';
 import {t, tct} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
-import {Organization} from 'sentry/types';
+import {Group, IssueCategory, Organization} from 'sentry/types';
 import {Event} from 'sentry/types/event';
 import trackAdvancedAnalyticsEvent from 'sentry/utils/analytics/trackAdvancedAnalyticsEvent';
 import {QuickTraceQueryChildrenProps} from 'sentry/utils/performance/quickTrace/types';
-import usePromptCheck from 'sentry/views/issueDetails/quickTrace/usePromptCheck';
+import useOrganization from 'sentry/utils/useOrganization';
 
 type Props = {
   event: Event;
+  group: Group;
   location: Location;
   organization: Organization;
   quickTrace: undefined | QuickTraceQueryChildrenProps;
-  isPerformanceIssue?: boolean;
 };
 
-function IssueQuickTrace({
-  event,
-  location,
-  organization,
-  quickTrace,
-  isPerformanceIssue,
-}: Props) {
-  const {shouldShowPrompt, snoozePrompt} = usePromptCheck({
-    projectId: event.projectID,
-    feature: 'quick_trace_missing',
-    organization,
-  });
+function PlaceholderEventNode({group}: {group: Group}) {
+  if (group.issueCategory !== IssueCategory.ERROR) {
+    return (
+      <EventNode type="black" icon={null}>
+        {t('This Event')}
+      </EventNode>
+    );
+  }
 
-  if (
-    !quickTrace ||
-    quickTrace.error ||
-    quickTrace.trace === null ||
-    quickTrace.trace.length === 0
-  ) {
-    if (!shouldShowPrompt) {
-      return null;
-    }
+  return (
+    <EventNode type="error" data-test-id="event-node">
+      <ErrorNodeContent>
+        <IconFire size="xs" />
+        {t('This Event')}
+      </ErrorNodeContent>
+    </EventNode>
+  );
+}
 
+function TransactionMissingPlaceholder({
+  type,
+  group,
+}: {
+  group: Group;
+  type?: QuickTraceQueryChildrenProps['type'];
+}) {
+  const organization = useOrganization();
+  useEffect(() => {
     trackAdvancedAnalyticsEvent('issue.quick_trace_status', {
       organization,
-      status: quickTrace?.type === 'missing' ? 'transaction missing' : 'trace missing',
-      is_performance_issue: isPerformanceIssue ?? false,
+      status: type === 'missing' ? 'transaction missing' : 'trace missing',
+      is_performance_issue: group.issueCategory === IssueCategory.PERFORMANCE,
```

Comment:
@malwilley we probably should pass in the `group.issueCategory` as `issue_category` here

Reply:
Agreed - I think we could actually get rid of this and add `trace_status` to the `issue_details.viewed` event instead? But I'd rather keep the analytics the same here and do that in a separate PR.

Reply:
@malwilley up to you, i think it's fine to throw it in here as well